src/blocks/notify.rs

```diff
@@ -54,21 +55,23 @@ pub async fn run(config: toml::Value, mut api: CommonApi) -> Result<()> {
     let config = NotifyConfig::deserialize(config).config_error()?;
     let mut widget = api
         .new_widget()
-        .with_format(config.format.with_default("")?);
+        .with_format(config.format.with_default(" $icon ")?);
 
     let mut driver: Box<dyn Driver + Send + Sync> = match config.driver {
         DriverType::Dunst => Box::new(MakoDriver::new().await?),
     };
 
     loop {
         let is_paused = driver.is_paused().await?;
+        let mut values = map!(
+            "icon" => Value::icon(api.get_icon(if is_paused { ICON_OFF } else { ICON_ON })?)
+        );
 
-        widget.set_icon(if is_paused { ICON_OFF } else { ICON_ON })?;
-        widget.set_values(if is_paused {
-            map!("paused" => Value::flag())
-        } else {
-            default()
-        });
+        if is_paused {
+            values.insert("paused".into(), Value::flag());
```

Comment:
Just a style thing, but this could be written as

```rust
let values = map!(
    "icon" => Value::icon(api.get_icon(if is_paused { ICON_OFF } else { ICON_ON })?),
    [if is_paused] "paused" => Value::flag(),
);
```